### ISSUE TITLE
Fractional style declaration values with the same whole part cause style name collision

### ISSUE THREAD

**Issue by mitrx** [roosta/herb#9]
Class names generated from data that differ only by the fractional parts of their values (e.g. 1, 1.1, 1.25, etc.) match and cause rule collision.

```Clojure
cljs.user> (require '[herb.core :refer [<class]])
nil
cljs.user> (defn icon-min-width-style [min-width] {:min-width min-width})
#'cljs.user/icon-min-width-style
cljs.user> (= (<class icon-min-width-style 1) (<class icon-min-width-style 1.25))
true
cljs.user> (= (<class icon-min-width-style 1) (<class icon-min-width-style 2.25))
false
```
Only the first rule with the class name gets injected and the following rules with the colliding selector get dropped.

This comes down to [hash generation for given style data](https://github.com/roosta/herb/blob/master/src/herb/impl.cljs#L167).
```Clojure
cljs.user> (= (hash {:style {:min-width 1}}) (hash {:style {:min-width 1.25}}))
true
cljs.user> (= (hash {:min-width 1}) (hash {:min-width 1.25}))
true
cljs.user> (= (hash 1) (hash 1.25))
true
```

**Comment by roosta** [comment]
So I do believe I fixed this issue in cfb1413, try out the new snapshot 
```clojure
[herb "0.10.1-SNAPSHOT"]
```

**Comment by mitrx** [comment]
The problem is gone with the snapshot version. Thank you very much.
